Connectors/Revit/Speckle.Connectors.RevitShared/HostApp/ElementUnpacker.cs

```diff
@@ -52,8 +41,12 @@ public IEnumerable<Element> UnpackSelectionForConversion(IEnumerable<Element> se
   /// </remarks>
   public IEnumerable<string> GetUnpackedElementIds(IEnumerable<string> objectIds)
   {
-    var doc = _revitContext.UIApplication?.ActiveUIDocument.Document!;
-    var docElements = doc.GetElements(objectIds);
+    var doc = revitContext.UIApplication?.ActiveUIDocument.Document;
+    var docElements = doc?.GetElements(objectIds);
+    if (docElements == null)
+    {
+      return [];
```

Comment:
Should we not throw if `Document` is null... the function didn't really Unpack the given Element ids... 
sounds like an exceptional case to me...

Reply:
Happy to chat on this. I've asked myself the same thing. Validating that `doc` is not `null` seems to happen a lot, like beforehand in the `RevitSendBinding`:

```
    var activeUIDoc =
      _revitContext.UIApplication.NotNull().ActiveUIDocument
      ?? throw new SpeckleException("Unable to retrieve active UI document");
```

or in the `RevitConversionSettingsFactory`:

`    var document = revitContext.UIApplication.NotNull().ActiveUIDocument.Document;`

That's possibly why it seems to be "ignored" here.